pyproject.toml

```diff
@@ -58,13 +56,11 @@ test = [
     "coverage[toml]",
     "import-linter==2.0",
     "ipylab>=1.0.0",
-    "ipython>=7.31.1, <8.0; python_version < '3.8'",
     "ipython~=8.10; python_version >= '3.8'",
```

Comment:
```suggestion
    "ipython~=8.10; python_version >= '3.9'",
```

Reply:
Nice catch!

Reply:
Or just 

```
"ipython~=8.10",
```

?

Reply:
Oh @astrojuanlu I think you are right. Let me remove the unnecessary constraints.